### ISSUE TITLE
Previous/Next button for search line edit

### ISSUE THREAD

**Issue by srazi** [pkulchenko/ZeroBraneStudio#824]
When I'm using mouse and I want to go previous or next search result I have to stop using mouse and press enter.

**Comment by pkulchenko** [comment]
@srazi, there is a left most icon on the search toolbar that goes to the next match when clicked (same behavior as `Enter`). If you click it when `Shift` is pressed, it goes to the previous result (same as `Shift-Enter`). Is this what you are looking for?

**Comment by srazi** [comment]
@pkulchenko Yes. Sorry, I just had tested it with `find in files`. In this mode (`find in files`) it just refresh search.

**Comment by pkulchenko** [comment]
What would you expect to happen with these buttons? I plan to have `F4`/`Shift-F4` shortcuts to navigate prev/next match, but the idea is that it will jump to the next result in the Results window and will open the file with the cursor positioned on the match line. Is that what you are looking for?

**Comment by srazi** [comment]
These buttons should do what  `F3/Shift-F3`. Indeed, I think `F3/Shift-F3` have an issue (feature?). When Result window has keyboard focus `F3/Shift-F3` find `previous/next` in Result window without a way to show selected result in Editor window.
